### PR TITLE
Correct URL for STF repository

### PR DESCRIPTION
**Pull request by keithc-ca**
The URL ends with `STF.git`, not `stf.git`.

### CODE REVIEW

File: jck/build.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 		<isset property="env.ADOPTOPENJDK_SYSTEMTEST_BRANCH"/>
 	</condition>
 
-	<condition property="stf_repo" value="${env.STF_REPO}" else="${git-prefix}://github.com/adoptium/stf.git">
+	<condition property="stf_repo" value="${env.STF_REPO}" else="${git-prefix}://github.com/adoptium/STF.gif">
```
Comment (llxia) [inline jck/build.xml:42]: It should be `git`, not `gif`. The same for the rest of the files.

Comment (keithc-ca) [reply jck/build.xml:42]: Oops, sorry, I'll fix that right away.

Comment (keithc-ca) [reply jck/build.xml]: Fixed.

Comment (llxia) [reply jck/build.xml]: Thanks :)

**Review by llxia — APPROVED** [review]
LGTM

**Comment by smlambert** [comment]
run aqa --sdk_resource nightly --build_list system --target _TestJlmRemoteThreadAuth --jdk_version 11 --jdk_impl hotspot --platform x86-64_linux

**Comment by keithc-ca** [comment]
The latest push should fix the builds which now explicitly specify the working directory when cloning STF.

**Comment by smlambert** [comment]
run aqa --sdk_resource nightly --build_list system --target _ConcurrentLoadTest_5m --jdk_version 11 --jdk_impl hotspot --platform x86-64_linux

**Comment by smlambert** [comment]
Thanks @keithc-ca !

**Pull request merged**